types/wordpress__block-editor/wordpress__block-editor-tests.tsx

```diff
@@ -44,26 +44,47 @@ const STYLES = [{ css: ".foo { color: red; }" }, { css: ".bar { color: blue; }",
 <be.BlockControls
     controls={[
         {
-            icon: "yes",
+            icon: "admin-appearance",
```

Comment:
@rafaucau here's my changes for the BlockControls tests, including updating attributes to current spec and adding a new test for nested arrays

Reply:
Thank you! Added it to my PR